can/interfaces/neousys/neousys.py

```diff
@@ -0,0 +1,272 @@
+""" Neousys CAN bus driver """
+
+#
+# This kind of interface can be found for example on Neousys POC-551VTC
+# One needs to have correct drivers and DLL (Share object for Linux) from Neousys
+#
+# https://www.neousys-tech.com/en/support-service/resources/category/299-poc-551vtc-driver
+#
+# Beware this is only tested on Linux kernel higher than v5.3. This should be drop in
+# with Windows but you have to replace with correct named DLL
+#
+
+# pylint: disable=R0903
+# pylint: disable=R0902
+# pylint: disable=C0413
+# pylint: disable=E0202
+# pylint: disable=W0611
+# pylint: disable=R1725
```

Comment:
This is probably easier to read:

```suggestion
# pylint: disable=too-few-public-methods
# pylint: disable=too-many-instance-attributes
# pylint: disable=wrong-import-position
# pylint: disable=method-hidden
# pylint: disable=unused-import
```

The last one, `R1725`/`super-with-arguments` is not required (see below).

Reply:
Why are the last two required? That seems odd ...